packages/lesswrong/lib/collections/curationNotices/views.ts

```diff
@@ -0,0 +1,15 @@
+import { ensureIndex } from '../../collectionIndexUtils';
+import CurationNotices from './collection';
+
+declare global {
+  interface CurationNoticesTerms extends ViewTermsBase {
+
+  }
```

Comment:
See `llmConversations/views.ts` for how to write more constrained view term types.  (Won't be a huge difference in your case, but it means any future additions will be correctly constrained.)

Reply:
Also I think the type should be called `CurationNoticesViewTerms`

Reply:
done (with robert)